packages/alice/lib/helper/alice_save_helper.dart

```diff
@@ -75,6 +52,22 @@ class AliceSaveHelper {
     }
   }
 
+  static Future<bool> _getPermissionStatus() async {
+    if (Platform.isAndroid || Platform.isIOS) {
+      return Permission.storage.status.isGranted;
+    } else {
+      return true;
+    }
+  }
+
+  static Future<bool> _requestPermission() async {
+    if (Platform.isAndroid || Platform.isIOS) {
```

Comment:
You should probably use [TargetPlatform](https://api.flutter.dev/flutter/foundation/TargetPlatform.html) here.